src/krux/pages/tools.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, ctx):
                     (t("Check SD Card"), self.sd_check),
                     (t("Print Test QR"), self.print_test),
                     (t("Create QR Code"), self.create_qr),
+                    (t("Wallet Addresses"), self.wallet_addresses),
```

Comment:
note: Tadeu and I discussed a renaming of this to  "Watch Only"... which is the mode a user will have in mind when using tools to review addresses (even if krux cannot watch anything, it seems as good a name as any other to me too).